asterius/src/Asterius/Builtins.hs

```diff
@@ -1239,6 +1260,31 @@ rtsMkHelper _ n con_sym = runEDSL n $ do
   storeI64 p 8 i
   emit p
 
+rtsMkBoolFunction _ = runEDSL "rts_mkBool" $ do
```

Comment:
There are already the same functions below. Maybe a rebase issue?

Reply:
Major rebase issue that broke stuff :( Will ping again when fixed